docs/Getting-Started.md

```diff
@@ -23,11 +24,12 @@ fastify.get('/', function (request, reply) {
 })
 
 // Run the server!
-fastify.listen(3000, function (err) {
+fastify.listen(3000, function (err, address) {
   if (err) {
     fastify.log.error(err)
     process.exit(1)
   }
+  console.log(`server listening on ${address}`)
```

Comment:
Use `fastify.log.info` instead of `console.log`.

Reply:
fixed